components/Cards/src/MDCCard.h

```diff
@@ -33,6 +33,18 @@
  */
 @property(nonatomic, readonly, strong, nonnull) MDCInkView *inkView;
 
+/**
+ This property defines if a card as a whole should be interactable or not.
+ What this means is that when isInteractable is set to NO, there will be no ink ripple and
+ no change in shadow elevation when tapped or selected.
+
+ Default is set to YES.
+
+ Important: Our specification for cards explicitly define a card as being an interactable component.
+ Therefore, this property should be set to NO *only if* there are other interactable items within
+ the card's content, such as buttons or other tappable controls.
+ */
+@property(nonatomic, assign) BOOL isInteractable;
```

Comment:
Naming is hard.

I would expect 'isInteractable' to enable / disable all user input on the card.

If the only effect of this is to disable the ripple, consider that as a name:
@property (nonatomic, assign, getter=isInkEnabled) BOOL inkEnabled;

Reply:
If we're disabling the Ripple (we should try to start distinguishing it from Ink for Material Theming), can't the client just set the color to clear? Also, I think the idea here is that the Card's actions will no longer function as a UIResponder. 

@yarneo Am I correct in my understanding of "Interactable"?

Reply:
It is more than just the ripple, it is shadow, the ripple, the stroke, etc.
Can we take a cue from UITableViewCell's UITableViewCellSelectionMode?
CardHighlightMode = Default || None

Should this interactivity be on highlight or selection?

Reply:
I have updated the PR that the tap will only apply to tappable subviews but not to the card container itself. This aligns better with the intractability notion, and will allow us to have the state not change as the tap isn't triggered. I think the name aligns well with the purpose of what we are trying to achieve with disallowing intractability, and also aligns better with the purpose of what userInteractionEnabled infers to the user.